Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -5,6 +5,65 @@ import XCTestDynamicOverlay
 
 // MARK: - Deprecated after 0.52.0
 
+extension EffectPublisher where Failure == Never {
+  @available(iOS, deprecated: 9999, message: "Use 'Effect.run' and pass the action to 'send'.")
+  @available(macOS, deprecated: 9999, message: "Use 'Effect.run' and pass the action to 'send'.")
+  @available(tvOS, deprecated: 9999, message: "Use 'Effect.run' and pass the action to 'send'.")
+  @available(watchOS, deprecated: 9999, message: "Use 'Effect.run' and pass the action to 'send'.")
+  public static func task(
```

Comment:
Should we add `Effect.task` and `Effect.fireAndForget` to the deprecation docs too?

Reply:
Done!